### PR TITLE
[DE] Add `Gameplay/Comboburst` and `Gameplay/Combo multiplier effect`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Combo_multiplier_effect/de.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+stub: true
+tags:
+  - score v1
+---
+
+# Combo-Multiplikatoreffekt
+
+Der **Combo-Multiplikatoreffekt** ist ein Problem bei [ScoreV1](/wiki/Gameplay/Score/ScoreV1), das erlaubt, dass der [Score](/wiki/Gameplay/Score) rückwärts zählt. Es handelt sich um einen Fehler bei der Darstellung von 32-Bit-Ganzzahlen mit Vorzeichen, bei der (in der Berechnung) die maximale Ganzzahl `2,147,483,647` beträgt. Sobald diese Grenze erreicht ist, zählt der Punktestand rückwärts. [ScoreV2](/wiki/Gameplay/Score#scorev2) behebt diese Problem, indem der Punktestand auf 1 Million Punkte begrenzt wird (Modifikationen nicht berücksichtigt).
+
+Der Combo-Multiplikatoreffekt tritt in [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko) und [osu!catch](/wiki/Game_mode/osu!catch) auf. Er kommt zustande, da die genannten Spielmodi die aktuelle [Combo](/wiki/Gameplay/Combo_(score_multiplier)) des Spielers als Teil der Berechnung des Scores verwenden. Das bedeutet, dass ein Spieler einen höheren Punktestand bekommt, wenn er eine [Full Combo (FC)](/wiki/Full_combo) bekommt, als jemand, der dieselbe Beatmap mit einer gebrochenen Combo gespielt hat.
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Combo_multiplier_effect/de.md:11]:
```suggestion
Der Combo-Multiplikatoreffekt tritt in [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko) und [osu!catch](/wiki/Game_mode/osu!catch) auf. Er kommt zustande, da die genannten Spielmodi die aktuelle [Combo](/wiki/Gameplay/Combo_(score_multiplier)) des Spielers als Teil der Berechnung des Scores verwenden. Das bedeutet, dass ein Spieler einen höheren Punktestand bekommt, wenn er eine [Full Combo (FC)](/wiki/Full_combo) erreicht, als jemand, der dieselbe Beatmap mit einer gebrochenen Combo gespielt hat.
```

**Review by The-Last-Cookie — COMMENTED** [review]
Suggestions from my side

File: wiki/Gameplay/Comboburst/de.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Combo-Bilder
+
+**Combo-Bilder** (auch *Combo-Bursts* genannt) lobpreisen die Erreichung eines [Combo-Meilensteins](/wiki/Gameplay/Combo_milestone) durch das kurzzeitige Aufblitzen eines Bildes. Diese Bilder sind in jedem Spielmodus unterschiedlich.
+
+- In [osu!](/wiki/Game_mode/osu!) und [osu!catch](/wiki/Game_mode/osu!catch) erscheinen Charaktere entweder von der linken oder rechten Seite des Fensters. Standardmäßig sind die erwähnten Charaktere [pippi](/wiki/Mascots#pippi) bzw. [Yuzu](/wiki/Mascots#yuzu).
+- In [osu!taiko](/wiki/Game_mode/osu!taiko) erscheint hinter [pippidon](/wiki/Mascots#don) eine Ansammlung von Blumen.
+- In [osu!mania](/wiki/Game_mode/osu!mania) erscheint standardmäßig ein Charakter an der rechten Seite der Bühne.
+
+In osu!, osu!taiko und osu!catch schießen für Combos, die kleiner als 100 sind, Sterne entweder aus der linken oder rechten Seite des Spielfeldes. Für Combos größer 100 schießen die Sterne aus beiden Seiten gleichzeitig heraus. In osu!mania schießen die Sterne aus der linken und rechten Seite der Bühne heraus.
```
Comment (FamousPig) [inline wiki/Gameplay/Comboburst/de.md:17]:
```suggestion
In osu!, osu!taiko und osu!catch schießen für Combos, die kleiner als 100 sind, Sterne entweder aus der linken oder rechten Seite des Spielfeldes. Für Combos über 100 schießen die Sterne aus beiden Seiten gleichzeitig heraus. In osu!mania schießen die Sterne aus der linken und rechten Seite der Bühne heraus.
```
Feels wrong to me, but I might just not have seen it before. Can't find any info either way.

**Review by FamousPig — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Comboburst/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Gameplay/Comboburst/de.md:17]: Sounds better 👍

**Review by FamousPig — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**